back/api/migrations/0039_alter_product_geom.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.12 on 2022-04-04 06:32
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0038_metadatacontact_is_validator'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="ALTER TABLE product ADD COLUMN geom2 geometry(MultiPolygon,2056);",
+            reverse_sql="ALTER TABLE product DROP COLUMN geom2;"
+        ),
+        migrations.RunSQL(
+            sql="CREATE INDEX IF NOT EXISTS product_geom2_id ON product USING gist (geom2)",
+            reverse_sql="DROP INDEX product_geom2_id;"
+        ),
+        migrations.RunSQL(
+            sql="UPDATE product set geom2 = ST_Multi(geom);",
+            reverse_sql="UPDATE product set geom = (ST_DUMP(geom2)).geom::geometry(Polygon,2056);"
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE product DROP COLUMN geom;",
+            reverse_sql="ALTER TABLE product ADD COLUMN geom geometry(Polygon,2056);"
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE product RENAME COLUMN geom2 TO geom;",
+            reverse_sql="ALTER TABLE product RENAME COLUMN geom TO geom2;"
+        ),
```

Comment:
```sql
ALTER TABLE product ALTER COLUMN geom type geometry(MultiPolygon, 2056) using ST_Multi(geom);
```

Wouldn't that be more concise? (maybe the reverse SQL would be quite difficult to get. I agree...)

Reply:
I didn't know this way with `using` keyword! Thanks!